doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -574,8 +572,8 @@ Requesting information
    .. cmdv:: Show Existentials
       :name: Show Existentials
 
-      It displays the set of all uninstantiated
-      existential variables in the current proof tree, along with the type
+      Displays all uninstantiated
+      existential variables in the current proof term, along with the type
```

Comment:
This is actually incorrect. See this example:
```
Goal 0 = 0.
eassert (_ = 0).
Show Existentials.
```

I agree nonetheless that this explanation would be worth rewording.

Reply:
Can you suggest better wording?  My change was just copy editing.  I don't know the feature that well.

Reply:
The issue here is rather that the terminology "existential variables" varies depending on context. According to the implementation, it is any `Evar` term. According to the [Gallina extension](https://coq.inria.fr/refman/language/gallina-extensions.html?highlight=existential#existential-variables) chapter, existential variables are what is written `?id`. The behavior of `Show Existentials` changed in 8.4, where it started to display `Evar`s in the sense of the new proof engine model (but indicating when the `Evar` is shelved or not), rather than what was formerly only dependent existential variables.

A possible updated wording could be: "Displays all open goals / existential variables of the current proof, along with their type".

In passing, the beginning of the Tactic chapter looks very obsolete. It could probably be merged with the beginning of the Proof handling chapter. Both should be rewritten almost from scratch actually with a modern eye (I don't know if, @zimmi48, or @jfehrle, you feel you would like to try to do that?). In particular, the existence of a "shelf" and of a "focus" should be explained early, as well as the fact that goals and existential variables are two incarnations of the same technical notion of "a hole of some type to fill".

Reply:
Indeed, I agree with your points, and we do have plans of refactoring the manual to solve this kind of issues.